src/index.ts

```diff
@@ -119,7 +133,11 @@ async function generateAndSetToken({ config, cache }: SalesforcePluginMeta): Pro
 
 export async function setupPlugin(meta: SalesforcePluginMeta) {
     verifyConfig(meta)
-    try{ 
+
+    const debugLoggingOn = meta.config.debugLogging === 'debug logging on'
+    logger = makeLogger(debugLoggingOn)
```

Comment:
This doesn't work in a plugins context. Global variables are not a thing. You should use `global` from the plugin meta.

As in:

```js
global.logger = makeLogger(debugLoggingOn)
```

And use `global.logger` everywhere.

Reply:
👍 

Was thinking of plugins like AWS lambda. Good learning. Thanks